Tests/SwiftSyntaxTest/BumpPtrAllocatorTests.swift

```diff
@@ -0,0 +1,39 @@
+import XCTest
```

Comment:
This file is missing the copyright header

Reply:
So as other test files in this repo. If we want them, let's do it in a separate PR.

Reply:
Sounds good. I think adding these headers would be a good thing to do. I don’t see a reason why some of these files shouldn’t contain them.